pkg/e/code.go

```diff
@@ -8,10 +8,12 @@ import (
 const (
 	// ErrorCodeConfigParseError is that an error occurs when it parse the file.
 	ErrorCodeConfigParseError ErrorCode = "config_parse_error"
+	// ErrorCodeConfigRegexpError is the regexp(re2) is invalid.
+	ErrorCodeConfigRegexpError ErrorCode = "config_regexp_error"
```

Comment:
Add a new error `config_regexp_error` to describe an error from the `regexp.MatchString`